net/snort3/patches/110-packet_capture-Fix-compilation-with-GCC-13.patch

```diff
@@ -18,5 +18,5 @@ src/network_inspectors/packet_capture/packet_capture.h:25:54: error: 'int16_t' d
  #include <string>
 +#include <cstdint>
```

Comment:
This patch is wrong since https://github.com/openwrt/packages/commit/65f6fee7c00a2afbe509247eb612f7d0c9afa16a

Where there is double include for `#include <cstdint>`? 🤔 

Already fixed in upstream in https://github.com/snort3/snort3/commit/70b811ca111f6df9d1353bda67e8383f69fcc5b0

Reply:
Plz check latest commit

Reply:
You need to push it first.

Reply:
Thought I did :smile:

Reply:
Sorry, I updated the wrong branch. Plz check now.

Reply:
Patch removed, but there isnt anything in the commit description.

Reply:
https://github.com/openwrt/packages/commit/bd7aa2ac44c780ca5e9c381525fc86d903df6e0e